src/main/kotlin/me/misik/api/domain/ReviewStyleService.kt

```diff
@@ -0,0 +1,10 @@
+package me.misik.api.domain
+
+import org.springframework.stereotype.Service
+
+@Service
+class ReviewStyleService {
+    fun getAll(): Array<ReviewStyle> {
+        return ReviewStyle.values()
```

Comment:
아래처럼 entries를 사용하는게 좋습니다!
kotlin에서도 추천하는 방식이고, values를 사용하면 매번 enum을 담은 새로운 Array를 생성해요 (entries는 캐싱된 값을 반환합니다)

```suggestion
        return ReviewStyle.entries
```